src/tools/manifest2proto.ts

```diff
@@ -32,3 +35,62 @@ export async function serialize2proto(path: string): Promise<Uint8Array> {
 
   return manifestProto.encode(message).finish();
 }
+
+const opts = minimist(process.argv.slice(2), {
+  string: ['outdir', 'outfile'],
+  alias: {d: 'outdir', f: 'outfile'},
+  default: {outdir: '.'}
+});
+
+if (opts.help || opts._.length === 0) {
+  console.log(`
+Usage
+  $ tools/sigh manifest2proto [options] path/to/manifest.arcs
+
+Description
+  Serializes manifests to a protobuf file. 
+
+Options
+  --outfile, -f output filename; required
+  --outdir, -d  output directory; defaults to '.'
+  --help        usage info
+`);
+  process.exit(0);
+}
+
+if (!opts.outfile) {
+  console.error(`Parameter --outfile is required.`);
+  process.exit(1);
+}
+
+// TODO(alxr): Support proto generation from multiple manifests
+if (opts._.length > 1) {
+  console.error(`Only a single manifest is allowed`);
+  process.exit(1);
+}
+
+if (opts._.some((file) => !file.endsWith('.arcs'))) {
+  console.error(`Only Arcs manifests ('*.arcs') are allowed.`);
+  process.exit(1);
+}
+
+async function main() {
+  try {
+    Runtime.init('../..');
+    fs.mkdirSync(opts.outdir, {recursive: true});
+
+    const buffer = await serialize2proto(opts._[0]);
+
+    const outPath = path.join(opts.outdir, opts.outfile);
+    console.log(outPath);
+
+    const outFile = fs.openSync(outPath, 'w');
+    fs.writeSync(outFile, Buffer.from(buffer));
+    fs.closeSync(outFile);
+  } catch (e) {
+    console.error(e);
+    process.exit(1);
+  }
+}
+
+void main();
```

Comment:
I would expect this to be a problem in a test - as test is importing this file for the sake of testing "serialize2proto", to sigh test still work with importing the CLI like that?

I've hit the same problem when working on the bundle tool, which is why I've put the core functionality into bundle.ts and the CLI wrapper into bundle-cli.ts, that's at least one solution. Thoughts?

Reply:
I like having those distinctions. If I may bikeshed on the naming conventions: 
Can we move `serialize2proto` to a new file and have the CLI be named `manifest2proto` to fit the other build scripts convention?

Reply:
... after looking at the code, I like the `-CLI.ts` breakdown.

Reply:
I am easy on naming here, whatever you think makes sense here. We can rename later if you wish.